src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -92,16 +99,30 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $config = $this->getConfig($extension, $container, $input->getOption('resolve-env'));
 
+        $format = $input->getOption('format');
+
+        if ('yaml' === $format && !class_exists(Yaml::class)) {
+            $errorIo->error('Setting the "format" option to "yaml" requires the Symfony Yaml component. Try running "composer install symfony/yaml" or use "--format=json" instead.');
+
+            return 1;
+        }
+
         if (null === $path = $input->getArgument('path')) {
             $io->title(
                 sprintf('Current configuration for %s', $name === $extensionAlias ? sprintf('extension with alias "%s"', $extensionAlias) : sprintf('"%s"', $name))
             );
 
-            $io->writeln(Yaml::dump([$extensionAlias => $config], 10));
+            $io->writeln($this->convertToFormat([$extensionAlias => $config], $format));
 
             return 0;
         }
 
+        if ('yaml' !== $format) {
+            $errorIo->error('The "path" option is only available for the "yaml" format.');
```

Comment:
Why? I don't see the issue to render a subsection of the data in JSON, is there?

Reply:
Condition took from another command, but indeed it's not a problem here